src/ontology/hp-edit.owl

```diff
@@ -154860,7 +154860,7 @@ SubClassOf(<http://purl.obolibrary.org/obo/HP_0030778> <http://purl.obolibrary.o
 
 # Class: <http://purl.obolibrary.org/obo/HP_0030779> (Ethmocephaly)
 
-AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://orcid.org/0000-0002-0003-6754>) Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "PMID:23248551") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/HP_0030779> "Ethmocephaly is the rarest form of holoprosencephaly, which occurs due to an incomplete cleavage of the forebrain. Clinically, the disease presents with a proboscis, hypotelorism, microphthalmos and malformed ears.")
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://orcid.org/0000-0002-0003-6754>) Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "PMID:23248551") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/HP_0030779> "Ethmocephaly is the rarrest form of holoprosencephaly, which occurs due to an incomplete cleavage of the forebrain. Clinically, the disease presents with a proboscis, hypotelorism, microphthalmos and malformed ears.")
```

Comment:
This correction is incorrect

Reply:
thanks for catching, there were some manual steps that caused some errors like this. should be fixed now :+1: